test/client-test.js

```diff
@@ -43,6 +43,7 @@ describe('Client', () => {
       it('returns SMART OAuth URIs', async function () {
         nock(this.baseUrl)
           .matchHeader('accept', 'application/json+fhir')
+          .matchHeader('Authorization', '')
```

Comment:
I don't think we send a value-less header here.

Reply:
This assertion would still pass, in `origin/master`

Reply:
okay, then let's 👍  this PR and fix it later.